src/components/button/style.ts

```diff
@@ -53,7 +53,7 @@ export const StyledButton = styled.button`
 
   &:hover {
     color: ${(props: any): string =>
-      props.mode === ThemeMode.dark ? props.theme.colors.highlight : props.theme.colors.hover};
+      props.color || (props.mode === ThemeMode.dark ? props.theme.colors.highlight : props.theme.colors.hover)};
```

Comment:
Should there be support for themes?

Reply:
he just added extra property for change behavior on hover, and support for the theme was there before

Reply:
I've just discussed it with Sasha. Currently, we use only light mode in all our designs, so he advised leaving it like this until the real use case appears.